test/unit/org/apache/cassandra/db/ColumnFamilyStoreTest.java

```diff
@@ -511,6 +516,41 @@ private void assertRangeCount(ColumnFamilyStore cfs, ColumnDefinition col, ByteB
         assertEquals(count, found);
     }
 
+    @Test
+    public void testSnapshotWithoutFlushWithSecondaryIndexes() throws Exception
+    {
+        String keyspaceName = KEYSPACE1;
+        String cfName= CF_INDEX1;
```

Comment:
These two vars seem redundant. We only use them once in the immediately following lines.